flash/core/utilities/imports.py

```diff
@@ -28,53 +30,6 @@
     Version = None
 
 
-def _module_available(module_path: str) -> bool:
-    """Check if a path is available in your environment.
-
-    >>> _module_available('os')
-    True
-    >>> _module_available('bla.bla')
-    False
-    """
-    try:
-        return find_spec(module_path) is not None
-    except AttributeError:
-        # Python 3.6
-        return False
-    except ModuleNotFoundError:
-        # Python 3.7+
-        return False
-    except ValueError:
-        # Sometimes __spec__ can be None and gives a ValueError
-        return True
-
-
-def _compare_version(package: str, op: Callable, version: str, use_base_version: bool = False) -> bool:
-    """Compare package version with some requirements.
-
-    >>> _compare_version("torch", operator.ge, "0.1")
-    True
-    >>> _compare_version("does_not_exist", operator.ge, "0.0")
-    False
-    """
-    try:
-        pkg = importlib.import_module(package)
-    except (ImportError, DistributionNotFound):
-        return False
-    try:
-        if hasattr(pkg, "__version__"):
-            pkg_version = Version(pkg.__version__)
-        else:
-            # try pkg_resources to infer version
-            pkg_version = Version(pkg_resources.get_distribution(package).version)
-    except TypeError:
-        # this is mocked by Sphinx, so it should return True to generate all summaries
-        return True
-    if use_base_version:
-        pkg_version = Version(pkg_version.base_version)
-    return op(pkg_version, Version(version))
-
-
 _TORCH_AVAILABLE = _module_available("torch")
 _PL_AVAILABLE = _module_available("pytorch_lightning")
 _BOLTS_AVAILABLE = _module_available("pl_bolts") and _compare_version("torch", operator.lt, "1.9.0")
```

Comment:
```suggestion
_TORCH_AVAILABLE = module_available("torch")
_PL_AVAILABLE = module_available("pytorch_lightning")
_BOLTS_AVAILABLE = module_available("pl_bolts") and compare_version("torch", operator.lt, "1.9.0")
```

Can you please change the names to `module_available` and `compare_version` in this file? :)